apps/application-system/api/src/app/modules/application/tools/applicationAccess.service.ts

```diff
@@ -8,8 +8,15 @@ import {
   ApplicationService,
 } from '@island.is/application/api/core'
 import { ApplicationTemplateHelper } from '@island.is/application/core'
-import { Application, ApplicationTypes } from '@island.is/application/types'
+import {
+  Application,
+  ApplicationContext,
+  ApplicationStateSchema,
+  ApplicationTemplate,
+  ApplicationTypes,
+} from '@island.is/application/types'
 import { getApplicationTemplateByTypeId } from '@island.is/application/template-loader'
+import { EventObject } from 'xstate'
```

Comment:
Does this pass format with no line between imports and implementation?

Reply:
Formatted before pushing so it should